Marlin/src/gcode/calibrate/M48.cpp

```diff
@@ -240,7 +240,7 @@ void GcodeSuite::M48() {
       sigma = SQRT(dev_sum / (n + 1));
 
       if (verbose_level > 1) {
-        SERIAL_ECHO((int)(n + 1));
+        SERIAL_ECHO(n + 1);
         SERIAL_ECHOPAIR(" of ", (int)n_samples);
```

Comment:
is int cast really needed anymore?

Reply:
I'm not sure. Currently, SERIAL_ECHOPAIR does not fallback to Serial.print directly but to `serial_echopair_PGM` and there is no overload for `uint8_t` so which one of the other overload will be selected by the compiler isn't clear. When cast, it's clear which one will be used. If I write the type punning version, I'll also remove the cast, as it'll be clear in all cases.